lib/gateway/Shard.js

```diff
@@ -548,7 +548,7 @@ class Shard extends EventEmitter {
                 */
                 this.emit("messageReactionAdd", message || {
                     id: packet.d.message_id,
-                    channel: channel || { id: packet.d.channel_id }
+                    channel: channel || {id: packet.d.channel_id}
```

Comment:
This needs typings changes. I saw it in another PR but forgot to comment on that that this breaks quite a bit of typings

Reply:
Could you fix that in another PR as this one is purely styling?

Reply:
for reference, introduced in #672